src/app/model-shared/types.ts

```diff
@@ -11,7 +11,18 @@ export interface HasAsyncGetTheValue {
   getTheValue(): Promise<string>;
 }
 
-
+export class EmailColumn extends radweb.StringColumn {
```

Comment:
it would be better to determine the caption here - and that way you won't need to send it in the constructor when it is used. 

You can also use the same EmailColumn - also in the Helpers Entity

Reply:
i did use it also in helpers :)
did not understand your comment on constructor

Reply:
I saw later that you've used it in helpers.

My comment regarding the constructor was that we instead of sending the work email in the constructor - I prefer that the EmailColumn will know that it's default caption is email  -so I won't need to specify it every where I use it.

I've fixed it already.